config/development.json

```diff
@@ -152,6 +152,7 @@
 		"publicize-preview": true,
 		"purchases/new-payment-methods": true,
 		"push-notifications": true,
+		"quake/github-integration-i1": true,
```

Comment:
I'd name it just `github-integration`. Feels weird to see a feature scoped by iteration and team.

Reply:
Sure, I changed it with d85f3d9284